### PR TITLE
doc: remove needless bindings

### PR DESCRIPTION
**Pull request by tshepang**
The extra syntax is more noise than help in simple examples like this

### CODE REVIEW

File: src/librustc_unicode/char.rs [hunk 1]
```diff
@@ -495,16 +476,11 @@ impl char {
...
-    ///
-    /// let c = '京';
-    /// assert!(c.is_alphabetic());
+    /// assert!('a'.is_alphabetic());
+    /// assert!('京'.is_alphabetic());
     ///
-    /// let c = '💝';
     /// // love is many things, but it is not alphabetic
-    /// assert!(!c.is_alphabetic());
+    /// assert!(!'❤️'.is_alphabetic());
```
Comment (steveklabnik) [inline src/librustc_unicode/char.rs]: changing my heart, tisk tisk! :wink:

**Comment by steveklabnik** [comment]
So, I think I feel good about this change, but I'm not sure. @GuillaumeGomez , what do you think?

**Comment by GuillaumeGomez** [comment]
In this case, it's obvious we're working on `char`s so the binding seems useless for me as well. I'm all for it! :)

**Comment by steveklabnik** [comment]
The heart change is actually breaking; :sparkling_heart: is a `char`, but :heart: isn't.

**Comment by eddyb** [comment]
@steveklabnik They both are part of Unicode, but @tshepang actually had `"\u{2764}\u{fe0f}"` - the second codepoint being "Variation Selector 16". It doesn't add the sparkles for me, but some OS with pervasive normalization might cause this on copy/paste.

**Comment by tshepang** [comment]
I reverted the weird heart thing

**Comment by steveklabnik** [comment]
travis is happy so i am happy. Thanks @tshepang !

@bors: r+ rollup

**Pull request merged**